ember/app/components/settings-panels/choose-club.js

```diff
@@ -30,7 +30,9 @@ export default Component.extend({
 
   saveTask: task(function * () {
     let club = this.get('club');
-    let json = { clubId: club.id };
+    let json;
+
+    json = { clubId: club.id || null };
```

Comment:
from https://github.com/skylines-project/skylines/issues/622 it seems like `club` is `undefined`, not `club.id` so fixing this with `|| null` most likely won't work. I think something like `clubId: club ? club.id : null` should work though.

side question: why did you split declaration and initialization of the variable?